jsnlog/PublicFacing/AspNet4/LogRequestHandling/HttpHandler/LoggerHandler.cs

```diff
@@ -25,14 +26,14 @@ namespace JSNLog
     // http://stackoverflow.com/questions/7247409/regarding-the-usage-of-irequiressessionstate
     // http://stackoverflow.com/questions/8039014/irequiressessionstate-vs-ireadonlysessionstate
 
-    public class LoggerHandler : IHttpHandler, IReadOnlySessionState
```

Comment:
I'm not sure but this might be breaking. We might need to duplicate this class and do the changes there, `[Obsolete]` the `LoggerHandler` and make it's implementation blocking